internal/cmd/destinations.go

```diff
@@ -87,15 +85,13 @@ func newDestinationsRemoveCmd() *cobra.Command {
 				return err
 			}
 
-			if destinations.Count == 0 {
```

Comment:
I still think this is a nice thing to be notified of

Reply:
This will be in a follow up where it will error if no matches are found unless a flag is supplied. In the meantime, this is bringing the destinations remove command in line with other commands